.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2
+dependencies:
+  cache_directories:
+    - ~/.composer/cache
+    - ~/build/vendor
+    - ~/build/node_modules
+jobs:
+  build:
+    docker:
+      - image: pantheonsystems/documentation:circle
+    working_directory: ~/build
+    steps:
+      - checkout
+      - run:
+          name: Generate build
+          command: |
+            npm install
+            ./node_modules/.bin/grunt
+            /documentation/bin/sculpin generate --env=prod --quiet
+      - run:
+          name: Deploy Multidev environment
+          command: bash ./scripts/deploy-multidev.sh
+      - run:
+          name: Start Ghost Driver
+          command: phantomjs --webdriver=8643
+          background: true
+      - run:
+          name: Start Sculpin
+          command: /documentation/bin/sculpin server
+          background: true
```

Comment:
It seems odd that some tests run against a local sculpin site and some run against the remote site. But let's not block the Circle 1 -> Circle 2 migration on that.

Reply:
I believe that the only tests run against the remote environment are the speedtests, where it makes sense to test the speed on the platform.